src/Scripting/CoreTest/TestCompilationFactory.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.VisualBasic;
```

Comment:
I ran into this myself earlier today - apparently, our pattern is to put them in Core (per @tmat).  Can you please add "CSharp" to the names so we know which ones are which?